finagle-zipkin-core/src/main/scala/BUILD

```diff
@@ -0,0 +1,21 @@
+scala_library(name='scala',
+  provides = scala_artifact(
+    org = 'com.twitter',
+    name = 'finagle-zipkin-core',
+    repo = artifactory,
+  ),
+  dependencies=[
+    '3rdparty/jvm/com/fasterxml/jackson/core:jackson-core',
+    '3rdparty/jvm/com/fasterxml/jackson/core:jackson-databind',
+    '3rdparty/jvm/com/fasterxml/jackson/module:jackson-module-scala',
+    '3rdparty/jvm/org/apache/thrift:thrift-0.5.0',
```

Comment:
one option is to drop this dep and instead use io.zipkin.java:zipkin:1.0.0 as that includes an implicit codec. In other words change to use the canonical model library.